media/js/firefox/new/scene1.js

```diff
@@ -60,7 +60,8 @@
             if (client.isFirefoxDesktop) {
                 client.getFirefoxDetails(function(data) {
                     // data.accurate will only be true if UITour API is working.
-                    if (data.channel === 'release' && data.isUpToDate && data.accurate) {
+                    // bug 1301721 only use major Firefox version until 49.0 is released
+                    if (data.channel === 'release' && client._isFirefoxUpToDate(false) && data.accurate) {
```

Comment:
Come to think of it, we probably don't need the additional check in here at all (showing it for the major release version number is consistent with the rest of the page logic). I'll just remove it.